src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/SessionSummaryService.kt

```diff
@@ -26,15 +26,19 @@ class SessionSummaryService(
   private val inductionScheduleService: InductionScheduleService,
 ) {
 
-  val today: LocalDate = LocalDate.now()
-
   /**
    * returns due, overdue and on hold counts for Inductions and Reviews
    * for all prisoners within a given prison.
    */
   fun getSessionSummaries(prisonId: String): SessionSummaryResponse {
     val prisoners = prisonerSearchApiService.getAllPrisonersInPrison(prisonId)
-    val prisonerNumbers = prisoners.map { it.prisonerNumber }
+      .also {
+        log.debug { "${it.size} prisoners returned from prisoner-search-api" }
+      }
+    val prisonerNumbers = prisoners.map { it.prisonerNumber }.toSet().sorted().toList()
```

Comment:
Some debug logging, but also remove any duplicates from the prisoner search list (this is part of how I found the root cause of this bug)